### ISSUE TITLE
Running Turtlebot2 on ROS Melodic with Ubuntu 18.04

### ISSUE THREAD

**Issue by jmvega** [JdeRobot/RoboticsAcademy#592]
Turtlebot2 is no longer supported on ROS Melodic.

**Comment by jmplaza** [comment]
Hi @jmvega could you please record a demo video about it? Is this issue closeable then?

**Comment by fqez** [comment]
Hi @jmvega I managed to make it work long time ago with ROS Melodic and ubuntu 18.04. I made a script that install the necessary pieces to make it work. Check it out, maybe you find it useful in some way :)

https://github.com/JdeRobot/base/tree/master/src/drivers/turtlebot_ws

**Comment by trunc8** [comment]
Hi @jmplaza, is this issue still open?

**Comment by jmplaza** [comment]
Hi @trunc8 , do you have access to a real Turtlebot2?

**Comment by trunc8** [comment]
Sorry, I do not have access to physical Turtlebot2 hardware, but I have worked with Turtlebot2 in Kinetic before. I can perform any software tests if that can be of any help!

**Comment by jmplaza** [comment]
Hi, 
no, this issue is just for tests with the real robot...

**Comment by jmplaza** [comment]
AFAIK @Carlosalpha1 you are already working on this, not for ROS-Noetic, but for ROS2-Foxy. Aren't you?

**Comment by Carlosalpha1** [comment]
Yes, @jmplaza, in a first approximation, I have the integration of kobuki_keyop for ROS2-Foxy [**here**](https://hub.docker.com/r/carlosalpha1/kobuki_keyop)
